frontend/packages/dev-console/src/components/guided-tour/GuidedTourText.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore: FIXME missing exports due to out-of-sync @types/react-redux version
+import { useSelector } from 'react-redux';
+import { PlusCircleIcon } from '@patternfly/react-icons';
+import { RootState } from '@console/internal/redux';
+import { useOpenshiftVersion } from '@console/shared/src/hooks/version';
+import { K8sResourceCommon } from '@console/internal/module/k8s';
+
+const DevPerspectiveTourText: React.FC = () => {
+  const openshiftVersion = useOpenshiftVersion();
+  return (
+    <>
+      Welcome to OpenShift {openshiftVersion?.slice(0, 3) ?? '4.x'}. Get started with a tour of some
+      of the key areas in the Developer perspective that can help you complete your workflows and be
+      more productive.
+    </>
+  );
+};
+
+export const devPerspectiveTourText = <DevPerspectiveTourText />;
+
+export const perspectiveSwitcherTourText = (
+  <>
+    <p>
+      The Perspective Switcher allows you to switch between the Developer and Admin perspectives.
+    </p>
+    <p>
+      Use the Admin perspective to manage workloads storage, networking, cluster settings, and more.
+      This may require additional user access.
+    </p>
+    <p>
+      Use the Developer perspective to build applications and associated components and services,
+      define how they work together, and monitor their health over time.
+    </p>
+  </>
+);
+
+export const searchTourText = (
+  <>
+    <p>
+      Search for resources in your project by simply starting to type or scrolling through a list of
+      existing resources.
+    </p>
+    <p>
+      Add frequently accessed resources to your side navigation for quick access. Look for the&nbsp;
+      <span style={{ color: 'var(--pf-global--palette--blue-400)' }}>
+        <PlusCircleIcon />
+        &nbsp;Add to navigation
```

Comment:
Use `{' '}` instead of `&nbsp;`. There are several instances of this.
```suggestion
        {' '}Add to navigation
```